module-os/board/rt1051/include/FreeRTOSConfig.h

```diff
@@ -182,9 +172,4 @@ standard names. */
 #define PROJECT_CONFIG_HEAP_INTEGRITY_CHECKS (1)
 #endif
 
-extern void vConfigureTimerForRunTimeStats(void);
-extern uint32_t ulHighFrequencyTimerTicks(void);
-#define portCONFIGURE_TIMER_FOR_RUN_TIME_STATS() vConfigureTimerForRunTimeStats()
-#define portGET_RUN_TIME_COUNTER_VALUE()         ulHighFrequencyTimerTicks()
-
```

Comment:
Nope, these definitions were moved to the board-specific `portmacro.h` to prevent clash with the existing ones from linux port. Actually, this change can be reverted as `FreeRTOSConfig.h` has been split along the development.

Reply:
but portmacro.h is deleted here? I can't see any added

Reply:
RT1051 `portmacro.h` is part of the FreeRTOS-kernel third-party module.

Reply:
ok - our configuration should be in our repository, not in externals - shouldn't it?

Reply:
Fixed.